src/app/(app)/integrations/reports/[reportId]/_components/stats-cards/malicious-components-card.tsx

```diff
@@ -0,0 +1,18 @@
+import { Card, CardHeader } from "@/components/ui/card";
+import { CardTitle } from "@/components/ui/card";
+import { CardContent } from "@/components/ui/card";
+
+export default function MaliciousComponentsCard({ count }: { count: number }) {
```

Comment:
why defining duplicate cards so many times. Instead just define once using props `title` and `count `and pass it in stats cards.